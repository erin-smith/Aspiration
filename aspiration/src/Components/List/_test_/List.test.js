import * as React from "react";
import {List, ListItem} from "../index";
import renderer from "react-test-renderer";

describe("List", () => {
  test("ListItem should match the snapshot", () => {
    const tree = renderer.create(<ListItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("List should match the snapshot", () => {
    const tree = renderer.create(<List/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("List with ListItems should match the snapshot", () => {
    const tree = renderer.create(<List><ListItem/><ListItem/></List>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
