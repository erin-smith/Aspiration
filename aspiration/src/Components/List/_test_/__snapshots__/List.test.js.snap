// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List List should match the snapshot 1`] = `
<div
  className="list-overflow-container"
>
  <ul
    className="list-group"
  />
</div>
`;

exports[`List List with ListItems should match the snapshot 1`] = `
<div
  className="list-overflow-container"
>
  <ul
    className="list-group"
  >
    <li
      className="list-group-item"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-xs-8 col-sm-9"
          >
            <h3 />
            <p />
          </div>
        </div>
      </div>
    </li>
    <li
      className="list-group-item"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-xs-8 col-sm-9"
          >
            <h3 />
            <p />
          </div>
        </div>
      </div>
    </li>
  </ul>
</div>
`;

exports[`List ListItem should match the snapshot 1`] = `
<li
  className="list-group-item"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-8 col-sm-9"
      >
        <h3 />
        <p />
      </div>
    </div>
  </div>
</li>
`;
